plugins/hazelcast3/src/main/resources/dist-sync-object-format.xml

```diff
@@ -22,10 +22,9 @@
         </join>
     </network>
 
-    <executor-service>
```

Comment:
The configuration changed, but I didn't have to remove it completely, so I'll put it back.